runtime/src/main/java/org/corfudb/runtime/CorfuRuntime.java

```diff
@@ -710,6 +710,7 @@ private EventLoopGroup getNewEventLoopGroup() {
                 parameters.nettyEventLoopThreads;
         ThreadFactory factory = new ThreadFactoryBuilder()
                 .setDaemon(true)
+                .setPriority (Thread.MAX_PRIORITY)
```

Comment:
Won't this lead to High priority threads even in the application as well?
Apart from that, state-transfer also uses CorfuRuntime to transfer data. If that uses high priority as well, we might end up in a worse situation where sequencer, log unit and layout threads get starved.